keylime/tpm/tpm_main.py

```diff
@@ -986,15 +986,15 @@ def __check_quote_c(self, pubaik, nonce, quoteFile, sigFile, pcrFile, hash_alg):
         retDict = self.__run(command, lock=False)
         return retDict
 
-    def check_quote(self, agent_id, nonce, data, quote, aikTpmFromRegistrar, tpm_policy={}, ima_measurement_list=None, allowlist={}, hash_alg=None, ima_keyring=None, mb_measurement_list=None, mb_refstate=None):
-        if hash_alg is None:
-            hash_alg = self.defaults['hash']
-
-        quoteFile = None
-        aikFile = None
-        sigFile = None
-        pcrFile = None
-
+    def do_tpm2_checkquote(self, aikTpmFromRegistrar, quote, nonce, hash_alg):
```

Comment:
Should this be named `_do_tpm2_checkquote` since it's not intended to be a public interface, right?

Reply:
I renamed it to `_tpm2_checkquote` now.